src/overlays/actors/ovl_En_Raf/z_en_raf.c

```diff
@@ -182,8 +182,8 @@ static DamageTable sDamageTable = {
  * Sets the `index`th pixel of the trap teeth texture to 0 (transparent black)
  * according to the `clearPixelTable`
  */
-void EnRaf_ClearPixelsTeeth(u16* texture, u8* clearPixelTable, s32 index) {
-    if ((index < 0x40) && (clearPixelTable[index] != 0)) {
+void EnRaf_ClearPixelTeeth(u16* texture, u8* clearPixelTable, s32 index) {
```

Comment:
I suppose the other thing to note here is that this function will *only* work with a texture that uses 16-bit pixels, hence why `texture` is a `u16*` here.